### ISSUE TITLE
Feature Request: Please add a grab download all subtitle button.

### ISSUE THREAD

**Issue by cx410** [AdvMaple/biliintl_subtitle_download_plugin#22]
Is it possible to add grab button all subtitles directly? later in the form of zip or rar

**Comment by AdvMaple** [comment]
Yes, but this will call a large number of api calls to Bilibli server, making you lose connection for about 2-5 minutes.

I can make this a little more efficient but this project is not my main priority at the moment. I will add this to my upgrade backlog.
